features/recipes/ingredients/operations.py

```diff
@@ -0,0 +1,189 @@
+from typing import List, Type
+
+from fastapi import HTTPException
+from sqlalchemy import update
+
+import db.connection
+from .exceptions import (
+    IngredientIntegrityViolation,
+    IngredientCategoryNotFoundException,
+    IngredientCategoryIntegrityViolation,
+    IngredientCategoryNameViolation,
+)
+from features.recipes.ingredients.models import Ingredient
+from .models import IngredientCategory
+from sqlalchemy.exc import IntegrityError
+
+
+def get_all_ingredients_category() -> List[Ingredient]:
```

Comment:
the function is called "get_all_ingredients_category" but return type is list of ingredients?